tests/Console/Command/ReadmeCommandTest.php

```diff
@@ -28,20 +29,33 @@
 {
     public function testIfReadmeFileIsCorrect()
     {
-        if (!class_exists('Symfony\Component\Console\Output\BufferedOutput')) {
-            $this->markTestSkipped('Unsupported symfony/console version, Symfony\Component\Console\Output\BufferedOutput was added in 2.4.');
-        }
+        $readmeFile = __DIR__.'/../../../README.rst';
+        $this->assertFileExists($readmeFile, sprintf('README file "%s" not found.', $readmeFile)); // switch to `assertFileIsReadable` on PHPUnit6
+        $this->assertTrue(is_readable($readmeFile), sprintf('Cannot not read "%s".', $readmeFile));
+        $this->assertTrue(is_file($readmeFile), sprintf('Expected file "%s".', $readmeFile));
+        $fileContent = file_get_contents($readmeFile);
+        $this->assertInternalType('string', $fileContent, sprintf('Failed to get content of "%s"', $readmeFile));
 
+        $app = new Application();
         $input = new ArrayInput(array('readme'));
         $output = new BufferedOutput();
-        $app = new Application();
-
-        $app->get('readme')->run($input, $output);
-
-        $fileContent = file_get_contents(__DIR__.'/../../../README.rst');
+        $output->setVerbosity(OutputInterface::VERBOSITY_VERY_VERBOSE);
+        $output->setDecorated(false);
+
+        $exitCode = $app->get('readme')->run($input, $output);
+        $output = $output->fetch();
+        // normalize line breaks, these are not important for the tests
+        $output = str_replace(PHP_EOL, "\n", $output);
+
+        $this->assertSame(
```

Comment:
This assertion can be done before fetching output, which would remove the need to use a variable for the exit code.